### ISSUE TITLE
Train in different epoch problem

### ISSUE THREAD

**Issue by chenhao324** [lyuwenyu/RT-DETR#241]
Hello author, I used the pytorch version of the code for fine-tuning. Why is the result obtained by the first epoch significantly better than that obtained by the second epoch

**Comment by lyuwenyu** [comment]
1. May be too little data, one epoch is enough.
2. Try adjusting the `learning rate`
3. ...

**Comment by chenhao324** [comment]
My data is over a million, and I also trained using pre trained weight parameters. I will try to modify the learning rate. Thank you for your answer
